src/controller/CHIPDeviceController.h

```diff
@@ -670,6 +677,11 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
         return mDefaultCommissioner == nullptr ? NullOptional : MakeOptional(mDefaultCommissioner->GetCommissioningParameters());
     }
 
+    // Reset the arm failsafe timer during commissioning.
+    void ExtendArmFailSafe(DeviceProxy * proxy, CommissioningStage step, uint16_t armFailSafeTimeout,
+                           Optional<System::Clock::Timeout> timeout, OnExtendFailsafeSuccess onSuccess,
```

Comment:
Please clearly document the difference between the two timeout params.

Reply:
updated parameter name to make it clear